### PR TITLE
Fix files

### PR DESCRIPTION
**Pull request by almonais**
image
add missing line on code example

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1100

File: packages/ng/libraries/date/src/lib/input/date-input.translate.ts [hunk 1]
```diff
@@ -26,6 +26,14 @@ export const luDateInputTranslations = {
 		formatMonth: 'MM/y',
 		formatYear: 'y',
 	},
+	'en-GB': {
+		placeholderDay: 'dd/mm/yyyy',
```
Comment (vvalentin-lucca) [inline packages/ng/libraries/date/src/lib/input/date-input.translate.ts:30]: Same format as in US, no?

**Comment by sanlucca** [comment]
image
Feels like your branch wasn't based on master :) Might need a rebase

**Comment by LuccaIntegration** [comment]
jenkins auto deploy http://lucca-front.lucca.local/PR-1100

**Pull request merged**